assets/app/views/directives/_status-icon.html

```diff
@@ -1,11 +1,13 @@
 <span ng-switch="status" class="hide-ng-leave">
+  <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
   <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
+  <span ng-switch-when="Deployed" class="fa fa-check text-success" aria-hidden="true"></span>
   <span ng-switch-when="Error" class="fa fa-times text-danger" aria-hidden="true"></span>
-  <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
-  <span ng-switch-when="Succeeded" class="fa fa-check text-success" aria-hidden="true"></span>
   <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true"></span>
-  <span ng-switch-when="Terminated" class="fa fa-times text-danger" aria-hidden="true"></span>
+  <span ng-switch-when="New" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
   <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
   <span ng-switch-when="Running" class="fa fa-refresh fa-spin" aria-hidden="true"></span>
+  <span ng-switch-when="Succeeded" class="fa fa-check text-success" aria-hidden="true"></span>
+  <span ng-switch-when="Terminated" class="fa fa-times text-danger" aria-hidden="true"></span>
```

Comment:
FWIW, I have this to disable the `RUNNING` animation: ` <status-icon status="build.status.phase" disableanimation></status-icon>.`  `NEW` and `PENDING` states use a built-in bootstrap class that mixes the look w/the behavior.  Screenshot provided, I think it is reasonable to assume the non-arrow spinner has no meaning without the animation effect & should not be disabled.

![screen shot 2015-11-23 at 1 35 32 pm](https://cloud.githubusercontent.com/assets/280512/11345753/17251af8-91e7-11e5-8f42-296e63dd3907.png)